cmd/kops/rollingupdatecluster.go

```diff
@@ -92,7 +92,7 @@ var (
 		# only roll the node instancegroup,
 		# use the new drain and validate functionality.
 		kops rolling-update cluster k8s-cluster.example.com --yes \
-		  --fail-on-validate-error="false" \
+		  --fail-on-drain-error="false" \
```

Comment:
I think the flag should be simply removed, since there is nothing in the preceding comment about not failing under a particular circumstance.

I also think you can remove the comment about "the new dran and validate functionality" since it isn't new anymore.